mobile/native/src/ln_dlc/mod.rs

```diff
@@ -117,6 +117,20 @@ const ON_CHAIN_SYNC_INTERVAL: Duration = Duration::from_secs(300);
 /// exact fee will be know.
 pub const FUNDING_TX_WEIGHT_ESTIMATE: u64 = 220;
 
+/// Triggers an update to the wallet balance and history, without an on-chain sync.
+pub async fn refresh_lightning_wallet() -> Result<()> {
```

Comment:
🔧 I think this should not be `async`. You'll have to wrap in `spawn_blocking` afterwards.